libraries/LittleFS/examples/SpeedTest/SpeedTest.ino

```diff
@@ -14,6 +14,24 @@
 // How large of a file to test
 #define TESTSIZEKB 512
 
+// Format speed in bytes/second.  Static buffer so not re-entrant safe
+const char *rate(long start, long stop, long bytes) {
+  static char buff[64];
+  if (stop == start) {
+    strcpy_P(buff, PSTR("Inf b/s"));
+  } else {
+    float r = 1000.0 * (float)bytes / (float)(stop - start);
+    if (r >= 1000000.0) {
```

Comment:
Compliant with how storage manufacturers count kB/MB/... etc ;)

Reply:
Hehe…take a guess where I’ve worked for the past twenty years.  :)